packages/api/AlvTimeWebApi/ErrorHandling/ErrorHandlingMiddleware.cs

```diff
@@ -23,6 +24,24 @@ public async Task Invoke(HttpContext context)
             {
                 await _next(context);
             }
+            catch (ValidationException ve)
+            {
+                context.Response.StatusCode = 400;
+                context.Response.ContentType = "application/json";
+
+                var problem = new ProblemDetails

```

Comment:
Nice! Should probably put the creation of the response in a separate method for reusability 😄